### PR TITLE
Create lab-wakeup.md

### PR DESCRIPTION
**Pull request by tahabi**
document more scripts for future staffers

### CODE REVIEW

File: ocfweb/docs/docs/staff/scripts/lab-wakeup.md [hunk 1]
```diff
@@ -0,0 +1,9 @@
+[[!meta title="lab-wakeup: wake up suspended hosts"]]
+
+## Introduction
+
+##### Usage: `lab-wakeup $HOST`
+
+Use `lab-wakeup $hostname` when you need to wake up a desktop that has suspended in order to SSH into it. For example, if you try to SSH into volcano.ocf.berkeley.edu and get an error similar to "ssh: connect to host $host port 22: No route to host" you can try `lab-wakeup volcano` on supernova or your staff VM to wake it, after which you should be able to log in without issue.
```
Comment (chriskuehl) [inline ocfweb/docs/docs/staff/scripts/lab-wakeup.md]: if not on supernova, probably need to install wakeonlan

File: ocfweb/docs/docs/staff/scripts/lab-wakeup.md [hunk 2]
```diff
@@ -0,0 +1,9 @@
+[[!meta title="lab-wakeup: wake up suspended hosts"]]
```
Comment (chriskuehl) [inline ocfweb/docs/docs/staff/scripts/lab-wakeup.md]: i'd `s/hosts/desktops/` to be clear

Comment (tahabi) [inline ocfweb/docs/docs/staff/scripts/lab-wakeup.md]: done

File: ocfweb/docs/docs/staff/scripts/lab-wakeup.md (hunk 1 above)
Comment (tahabi) [inline ocfweb/docs/docs/staff/scripts/lab-wakeup.md]: done

**Comment by jvperrin** [comment]
Looks good, might be a good place to mention [ocf-suspend](https://github.com/ocf/puppet/blob/master/modules/ocf_desktop/files/suspend/ocf-suspend) which does the suspending part of this on desktops and is run every 15 minutes. It also might be good to add to the [apt-dater docs](https://github.com/ocf/ocfweb/blob/master/ocfweb/docs/docs/staff/procedures/installing-updates.md) and mention that desktops should be woken up before installing any updates (and that it's good not to do this right before a 15 minute interval, otherwise the desktops might just sleep again).

**Comment by tahabi** [comment]
@jvperrin I'll amend the apt-dater docs. create a new file for ocf-suspend or just add it as a footnote to this one?

**Comment by jvperrin** [comment]
I'd just add it as a footnote to this one, since it isn't a script typically run by hand. It just seems like a good note to add to this file since suspending and resuming go together well.

**Comment by tahabi** [comment]
reference added in apt-dater docs as well. Merging this PR

**Pull request merged**